pygerm/client/__init__.py

```diff
@@ -11,26 +11,62 @@ def parse_event_payload(data):
     The layout is
 
        "0" [[4 bit chip addr] [5 bit channel addr]] [10 bit TD] [12 bit PD]
-       "1" [31 bit time stamp]
+       "1000" [28 bit time stamp]
 
     '''
 
     # TODO sort out if this can be made faster!
+    word1 = data[::2]
+    word2 = data[1::2]
 
     # chip addr
-    chip = (data >> (27)) & 0xf
+    chip = word1 >> 27 & 0xf
     # chan addr
-    chan = (data >> (22)) & 0x1f
+    chan = word1 >> 22 & 0x1f
     # fine ts
-    td = (data >> (12)) & 0x3ff
+    # TODO : why 0x3ff not 1ff??
+    td = word1 >> 12 & 0x1ff
+
     # evergy readings
-    pd = (data) & 0xfff
+    pd = word1 & 0xfff
+
     # FPGA tick
-    ts = data >> 32 & 0x7fffffff
+    # should be 6 f's not 7??
+    ts = word2 & 0x7ffffff
```

Comment:
In documentation from last time had the timestamp listed as 31 bits (not 28)

Reply:
![image](https://user-images.githubusercontent.com/705366/33025099-f3918892-cdda-11e7-977a-ac9a7bd72862.png)

Reply:
that's what i got, is that correct or changed?

Reply:
It has been changed 

![image](https://user-images.githubusercontent.com/199813/33025597-6f27617e-cddc-11e7-92d3-9f01d0860d0e.png)

Reply:
That is from a version of the docs from May.